ui/package.json

```
@@ -30,9 +30,9 @@
     "start:mirage": "start () { MIRAGE_DEV_HANDLER=$1 yarn run start; }; start",
     "test": "npm-run-all --print-name lint:js:quiet lint:hbs:quiet && node scripts/start-vault.js",
     "test:enos": "npm-run-all lint:js:quiet lint:hbs:quiet && node scripts/enos-test-ember.js",
-    "test:oss": "yarn run test -f='!enterprise'",
-    "test:quick": "node scripts/start-vault.js",
-    "test:quick-oss": "yarn test:quick -f='!enterprise'",
+    "test:oss": "yarn run test -f='!enterprise' --split=8 --parallel",
```

Something I found while playing with this branch is that passing the filter param no longer works because it expects a value for `--parallel` to come next. I think this can be fixed with: 
```suggestion
    "test:oss": "yarn run test -f='!enterprise' --split=8 --parallel=1",
```